CHANGES.md

```diff
@@ -1,5 +1,8 @@
 # ?? (??) - ??
 
+# 3.10.1 (2021-09-06) - Paris
```

Comment:
Why do we have a version here ? has there been a new release ?

Reply:
Sorry, my bad. This is a version number used for developement. I haven't paid close enough attention to the versioning practices of js_of_ocaml. (I'm used to projects having an opam file in the repository with a version that changes from PR to PR.)

Reply:
https://github.com/ocsigen/js_of_ocaml/pull/1128